tensorflow/core/platform/default/build_config.bzl

```diff
@@ -8,7 +8,7 @@ load("//tensorflow:tensorflow.bzl", "if_not_mobile")
 WITH_GCP_SUPPORT = False
 WITH_HDFS_SUPPORT = False
 WITH_XLA_SUPPORT = False
-WITH_JEMALLOC = True
+WITH_JEMALLOC = False
```

Comment:
A `bazel clean` and `bazel build` later, it seems to be okay with `WITH_JEMALLOC = True`, no interference on RPi.